Dockerfile.dapper

```diff
@@ -23,7 +23,7 @@ RUN go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest && \
     cp $ENVTEST_BIN/* /usr/local/kubebuilder/bin
 
 ENV GO111MODULE on
-ENV DAPPER_ENV REPO TAG DRONE_TAG CROSS GITHUB_TOKEN SKIP_TESTS
+ENV DAPPER_ENV REPO TAG DRONE_TAG CROSS GITHUB_TOKEN SKIP_TESTS GIT_TAG
```

Comment:
Is this something coming from some custom script? Not a big deal, we will drop Dapper one day.

Reply:
yeah, this is just to facilitate the dev testing, this is how I test my changes:

build_push.sh
```
#!/bin/bash
SKIP_TESTS=true GIT_TAG=dev REPO=husseingalal make
docker push husseingalal/k3k:dev
docker push husseingalal/k3k:k3k-kubelet
sleep 10
helm delete k3k -n k3k-system
helm install --namespace k3k-system --set image.repository=husseingalal/k3k --set image.tag=dev --set sharedAgent.image.repository=husseingalal/k3k --set sharedAgent.image.tag=k3k-kubelet k3k k3k/k3k --devel --set-string debug=true --set image.pullPolicy=Always --set sharedAgent.image.pullPolicy=Always
```

Reply:
Yes, I too have a "custom script" for my development flow.

Since I will try to create the Development documentation for the #202 (and #231) it's probably better to have something we agree, altough the real "local" development could still diverge a bit. :smile:

Reply:
we will eventually just remove the dapper altogether, this is just temporary